### PR TITLE
[UI] Add markdown of discription view

### PR DESCRIPTION
**Pull request by LeoSilvaGomes**
### Change the description client

- Update the TextField to can preview the markdown way, updated to MarkdownTextArea and putted on context

Issue: #1460

---

Result:

- before fix:

image

- after fix:

image

image

### CODE REVIEW

**Comment by djmitche** [comment]
Yes, #1473 should fix it.

**Comment by LeoSilvaGomes** [comment]
@helfi92 Done! :+1:

File: ui/src/components/ClientForm/index.jsx [hunk 1]
```diff
@@ -323,15 +328,13 @@ export default class ClientForm extends Component {
               }
             />
           </ListItem>
-          <ListItem>
-            <TextField
-              label="Description"
-              name="description"
+          <ListItem className={classes.clickDescriptionListItem}>
```
Comment (helfi92) [inline ui/src/components/ClientForm/index.jsx]: What does `click` in `clickDescriptionListItem` mean? What do you think if we omit `click` and instead rename it to `descriptionListItem`.

Comment (LeoSilvaGomes) [reply ui/src/components/ClientForm/index.jsx]: What do you think of changing clientDescriptionListItem? I thought about differentiating this description view from others.

Comment (helfi92) [reply ui/src/components/ClientForm/index.jsx]: Sure. `clientDescriptionListItem` sounds good to me :)

Comment (LeoSilvaGomes) [reply ui/src/components/ClientForm/index.jsx]: done!

**Pull request merged**